### ISSUE TITLE
`mintx unbond` panics

### ISSUE THREAD

**Issue by zramsay** [eris-ltd/mint-client#62]
`eris chains exec someChain "mintx unbond --help"`
- expected: help output
- got:
```
panic: shorthand redefinition

goroutine 1 [running]:
panic(0x7fdda0, 0xc8201306a0)
	/usr/lib/go/src/runtime/panic.go:481 +0x3e6
github.com/eris-ltd/mint-client/Godeps/_workspace/src/github.com/spf13/pflag.(*FlagSet).AddFlag(0xc820147d00, 0xc820146e00)
	/go/src/github.com/eris-ltd/mint-client/Godeps/_workspace/src/github.com/spf13/pflag/flag.go:506 +0xcae
github.com/eris-ltd/mint-client/Godeps/_workspace/src/github.com/spf13/cobra.(*Command).mergePersistentFlags.func1(0xc820146e00)
	/go/src/github.com/eris-ltd/mint-client/Godeps/_workspace/src/github.com/spf13/cobra/command.go:999 +0x31
github.com/eris-ltd/mint-client/Godeps/_workspace/src/github.com/spf13/pflag.(*FlagSet).VisitAll(0xc820146d80, 0xc820047738)
	/go/src/github.com/eris-ltd/mint-client/Godeps/_workspace/src/github.com/spf13/pflag/flag.go:223 +0x85
github.com/eris-ltd/mint-client/Godeps/_workspace/src/github.com/spf13/cobra.(*Command).mergePersistentFlags(0xc820144820)
	/go/src/github.com/eris-ltd/mint-client/Godeps/_workspace/src/github.com/spf13/cobra/command.go:1002 +0x237
github.com/eris-ltd/mint-client/Godeps/_workspace/src/github.com/spf13/cobra.stripFlags(0xc820130620, 0x1, 0x1, 0xc820144820, 0x0, 0x0, 0x0)
	/go/src/github.com/eris-ltd/mint-client/Godeps/_workspace/src/github.com/spf13/cobra/command.go:326 +0x8b
github.com/eris-ltd/mint-client/Godeps/_workspace/src/github.com/spf13/cobra.(*Command).Find.func1(0xc820144820, 0xc820130620, 0x1, 0x1, 0x6, 0x0, 0x0, 0x0)
	/go/src/github.com/eris-ltd/mint-client/Godeps/_workspace/src/github.com/spf13/cobra/command.go:386 +0x7d
github.com/eris-ltd/mint-client/Godeps/_workspace/src/github.com/spf13/cobra.(*Command).Find.func1(0xc820144d00, 0xc82000a3d0, 0x2, 0x2, 0x75, 0x0, 0x0, 0x0)
	/go/src/github.com/eris-ltd/mint-client/Godeps/_workspace/src/github.com/spf13/cobra/command.go:394 +0x2b5
github.com/eris-ltd/mint-client/Godeps/_workspace/src/github.com/spf13/cobra.(*Command).Find(0xc820144d00, 0xc82000a3d0, 0x2, 0x2, 0x4a1e6d, 0x0, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/eris-ltd/mint-client/Godeps/_workspace/src/github.com/spf13/cobra/command.go:416 +0x175
github.com/eris-ltd/mint-client/Godeps/_workspace/src/github.com/spf13/cobra.(*Command).Execute(0xc820144d00, 0x0, 0x0)
	/go/src/github.com/eris-ltd/mint-client/Godeps/_workspace/src/github.com/spf13/cobra/command.go:540 +0x11c
main.main()
	/go/src/github.com/eris-ltd/mint-client/mintx/main.go:183 +0x1492
Container interactive-46eb800b-9795-4ade-9aa2-d15a4e7e6384 exited with status 2
```

**Comment by zramsay** [comment]
hrm, see last comment in #33

**Comment by zramsay** [comment]
closed by #66
